include/deal.II/lac/scalapack.h

```diff
@@ -467,7 +467,7 @@ class ScaLAPACKMatrix : protected TransposeTable<NumberType>
    * and <tt>chunk_size.second</tt> the number of columns.
    */
   void
-  save(const char *                                 filename,
```

Comment:
since `save/load` made it to `9.0.0`, we usually mark old function deprecated and remove them in subsequent release. Do you mind adding something like this 

```
/**
      * Same as above but takes a filename as a <code>char</code> pointer.
      *
      * @deprecated This function is deprecated. Use the one which takes <code>std::string</code> instead.
      */
     DEAL_II_DEPRECATED void
     save(const char *                                 filename, ....);
```
and call the new one from this with `std::string(filename)` ?